src/fable/Fable.Compiler/Replacements.fs

```diff
@@ -169,11 +169,13 @@ module Util =
         | Fable.String ->
             GlobalCall ("Number", Some "parseInt", false, args)
             |> makeCall com i.range i.returnType
+            |> wrap i.returnType
         | Fable.Number kindFrom ->
             match i.returnType with
             | Fable.Number kindTo ->
                 let pattern = if needToCast kindFrom kindTo then patternFor kindTo else "$0"
                 emit i pattern [args.Head]
+                |> wrap i.returnType
```

Comment:
You're right, I took them out, I only added wrapping because it sounded right, there was no difference in the output either way.